napari/layers/labels/labels.py

```diff
@@ -374,24 +387,21 @@ def color(self, color):
         self.color_mode = color_mode
 
     def _ensure_int_labels(self, data):
-        """Ensure data is integer by converting from float if required"""
+        """Ensure data is integer by converting from bool if required, raising an error otherwise"""
```

Comment:
`"""Ensure data is integer by converting from bool if required, raising an error otherwise."""`

(Minor nitpick, docstrings are supposed to be treated as sentences)